protoc-gen-gogo/generator/generator.go

```diff
@@ -2071,124 +2071,129 @@ func (g *Generator) generateMessage(message *Descriptor) {
 		}
 	}
 
-	for i, field := range message.Field {
-		// Allocate the getter and the field at the same time so name
```

Comment:
Ok thats fine, then just bring back the comment :)

This is coming all the way from golang/protobuf so we want to fix it there and then pull that fix in here.